README.md

```diff
@@ -11,19 +11,32 @@ To submit a bug or suggest a documentation fix/enhancement for a specific page,
 ## Edit the site
 To edit a single page, navigate to the page, and click "Edit this page on GitHub" in the page footer. That will create a simple Pull Request where you can propose your changes.
 
-You can also clone and fork the repository like any other GitHub project, if you are more comfortable with that. See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on building and testing the site locally.
+You can also clone and fork the repository like any other GitHub project, if you are more comfortable with that.
+See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on building and testing the site locally.
 
 See [MAINTAINING.md](MAINTAINING.md) for information about site maintainers.
 
 See [README_ORGANIZATION.md](README_ORGANIZATION.md) for information about how the source code is organized.
 
-All content in the website is written in Markdown. For information on formatting Markdown, see [Markdown Basics](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/).
+Most of the content in the website is written in Markdown.
+For information on formatting Markdown, see [Markdown Basics](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/).

```

Comment:
For information on authoring in Asciidoc, see https://asciidoctor.org/docs/asciidoc-writers-guide/